airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -54,6 +57,22 @@ def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapp
         response_json = response.json()
         yield from response_json.get("data", [])  # Stripe puts records in a container array "data"
 
+    @staticmethod
+    def chunk_dates(start_date_ts: int) -> Iterable[Tuple[int, int]]:
+        now = pendulum.now().int_timestamp
+        step = int(pendulum.duration(days=30).total_seconds())
```

Comment:
I think this should be configurable, with a very large window by default since we don't want to negatively impact the majority for a minority of users having problems

Reply:
done